Changes after review (#86)
nf-core/seqsubmit: docs/usage.md

  - provide the path to a local database with `--checkm2_db`, otherwise the pipeline downloads the version specified with `--checkm2_db_download_id` (by default Zenodo accession `14897628`).

- CAT_pack:

>   - provide the path to a local database (containing `tax/` and `db/` folders or a `tar.gz` archive) with `--cat_db`, otherwise the pipeline constructs NCBI non-redundant protein database.

Reviewer: ```suggestion
  - provide the path to a local database (containing `tax/` and `db/` folders or a `tar.gz` archive) with `--cat_db`, otherwise the pipeline builds CAT_pack DB from NCBI non-redundant protein database (`nr`).
```

| -------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
| `--upload_tpa` | No       | Mark assemblies as [third party assemblies (TPA)](#third-party-assemblies-tpa) when required. Default: `false`.                                |
| `--checkm2_db` | No       | Path to a local CheckM2 database. If omitted, downloads the version set by `--checkm2_db_download_id`. Default: `14897628` (Zenodo accession). |

> | `--cat_db`     | No       | Path to a local CAT_pack database. If omitted, constructs the NCBI non-redundant protein database.                                             |

Reviewer: Same
```suggestion
| `--cat_db`     | No       | Path to a local CAT_pack database. If omitted, builds CAT_pack DB from NCBI non-redundant protein database (`nr`).                                             |
```
